content/en/docs/setup/platform-setup/kind/index.md

```diff
@@ -0,0 +1,119 @@
+---
+title: kind
+description: Instructions to setup kind for Istio.
+weight: 21
```

Comment:
I do use `make serve` and `make lint` locally and I think I have resolved most of the linting issues because of that. But definitely you are far more better and accurate than `linter`. Really appreciate your suggestions.  

I had a no idea about weighted numbers, but as you explain it better I think I understood it now.